feature-1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added the feature-1 branch
```

Comment:
It's really good that you have decided to add this line of code!

Reply:
Thanks for this comment (Bogdan)